src/MenuItem.js

```diff
@@ -64,8 +64,20 @@ class MenuItem extends React.Component {
 }
 
 MenuItem.propTypes = {
+
+  /**
+   * Highlight the menu item as active.
+   */
   active: React.PropTypes.bool,
+
+  /**
+   * Disable the menu item, making it unselectable.
+   */
   disabled: React.PropTypes.bool,
+
+  /**
+   * Make the menu item a divider.
```

Comment:
I might explain this a bit more, since using the word "divider" to describe `divider` doesn't add a lot of clarity. Perhaps something like: "Styles a menu item as a horizontal rule, providing visual separation between groupings of menu items"